### PR TITLE
/move command now shows the track that was moved and its position

### LINKED ISSUE #592: Enhancement - Add Song name to move command
Wouldn't it be better if the song name and the new position are mentioned in the reply message?

Tried a few things about this in my fork but can't get it to work. (maybe because I'm all along new to TS :D)

### PR DESCRIPTION
**Pull request by therithwikrayani**
Closes #592 

/move command now shows the track that was moved and its position

- [x] I updated the changelog

### CODE REVIEW

**Comment by therithwikrayani** [comment]
Sorry for all the linting phew

File: src/services/player.ts [hunk 1]
```diff
@@ -379,12 +379,14 @@ export default class {
...
 
-  move(from: number, to: number): void {
+  move(from: number, to: number): string {
     if (from > this.queueSize() || to > this.queueSize()) {
       throw new Error('Move index is outside the range of the queue.');
     }
 
     this.queue.splice(this.queuePosition + to, 0, this.queue.splice(this.queuePosition + from, 1)[0]);
+
+    return this.queue[this.queuePosition + to].title;
```
Comment (codetheweb) [inline src/services/player.ts:389]: Let's have this return the entire queued track instead (drop the `.title`) and then callers of this method can select what they want.

Comment (therithwikrayani) [reply src/services/player.ts]: done as requested

**Comment by therithwikrayani** [comment]
Kinda unrelated question: why did the linter not requre object destructuring for commit 08175a8 but needed it for c9a2904?

File: src/commands/move.ts [hunk 2]
```diff
@@ -40,8 +40,8 @@ export default class implements Command {
       throw new Error('position must be at least 1');
     }
 
-    player.move(from, to);
+    const {title} = player.move(from, to);
 
-    await interaction.reply('moved');
+    await interaction.reply('moved **' + title + '** to position **' + String(to) + '**');
```
Comment (Zagrthos) [inline src/commands/move.ts:45]: I would change this line to ``` await interaction.reply(`moved **${title}** to position **${String(to)}**`); ```
but this is just my preference :)

**Comment by codetheweb** [comment]
> Kinda unrelated question: why did the linter not requre object destructuring for commit https://github.com/codetheweb/muse/commit/08175a83c7692cf850c76ac06f0abd4c07c303ef but needed it for https://github.com/codetheweb/muse/commit/c9a29045182f2104daff67970faa041bc4334e1d?

The rule is looking for instances of `(const | let) <variable-name> = <expression>.<property>`. There's nothing to destructure in https://github.com/codetheweb/muse/commit/08175a83c7692cf850c76ac06f0abd4c07c303ef.

**Review by codetheweb — APPROVED** [review]

**Pull request merged**